lib/OpenQA/Worker/Settings.pm

```diff
@@ -27,8 +27,16 @@ sub new {
     $cli_options //= {};
 
     my $settings_file = ($ENV{OPENQA_CONFIG} || '/etc/openqa') . '/workers.ini';
-    $settings_file = undef unless -e $settings_file;
-    my $cfg = Config::IniFiles->new(-file => $settings_file);
+    my $cfg;
+    my @parse_errors;
+    if (-e $settings_file) {
+        $cfg = Config::IniFiles->new(-file => $settings_file);
+        push(@parse_errors, @Config::IniFiles::errors) unless $cfg;
+    }
+    else {
+        push(@parse_errors, "Config files does not exist under '$settings_file'.");
```

Comment:
I assume this should be referring to a single file:

`Config file not found at '$settings_file'.`

Reply:
So s/under/at/ or what do you suggest?